ip/flac.c

```diff
@@ -161,15 +159,6 @@ static FLAC__StreamDecoderWriteStatus write_cb(const Dec *dec, const FLAC__Frame
 	int ch, nch, i = 0;
 	char *dest; int32_t src;
 
-	if (ip_data->sf == 0) {
-		return FLAC__STREAM_DECODER_WRITE_STATUS_CONTINUE;
-	}
```

Comment:
ip_data->sf should be non zero after _open. if it's zero things are terribly wrong.